easybuild/easyconfigs/m/malt/malt-0.4.1.eb

```diff
@@ -0,0 +1,40 @@
+# This is for the custom build of Malt 
+# Malt is downloaded from here:
+# http://ab.inf.uni-tuebingen.de/data/software/malt/download/welcome.html
+# The MALT_unix_0_4_1.sh is executed and the binaries are installed in
+# malt/bin
+# The malt directory is bundled into a tarball which will be installed
+# This is a bit of a hack!
+# The homepage is here:
+# http://ab.inf.uni-tuebingen.de/software/malt/
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PackedBinary'
+
+name = 'malt'
+version = '0.4.1'
+
+homepage = 'http://ab.inf.uni-tuebingen.de/software/malt/'
+description = """MALT (MEGAN alignment tool) is an extension of MEGAN (metagenome analyzer).
+
+MALT performs alignment of metagenomic reads against a database of reference sequences 
+(such as NR, GenBank or Silva) and produces a MEGAN RMA file as output. The software is 
+currently under development. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['malt-%(version)s.tar.gz']
+checksums = ['68ac3c1b0a862a595eaa13ae72a49974292bcea486cf5fff531113993afd6b61']
+
+# specify dependency on Java/1.8 "wrapper", rather than a specific Java version
+dependencies = [('Java', '1.8', '', True)]
+
+# modloadmsg = "To execute %(name)s run: ImageJ-linux64"
```

Comment:
@sassy-crick Can you please clarify on this line?

Reply:
@vanzod Do you mean the line with the dependencies? If that is the line you are after, please see here:
https://easybuild.readthedocs.io/en/latest/Wrapping_dependencies.html
If you mean the # modloadmsg line, I really should have removed that. Sorry. 
If you have a better idea how to donwload the installer and get that working I am open for that. As I said, that is a bit of a hack and I am not too happy with that.

Reply:
The comment was made on the `modloadmsg` line, so please drop that. :)

Reply:
@boegel Thanks. It was not quite clear to me which line was meant here. I will upload the new version in a moment. :-)